analyze_visual/train.py

```diff
@@ -104,10 +105,6 @@ def data_preparation(x):
     # fit and transform the data
     x_all = scaler.fit_transform(x_all)

```

Comment:
This is the wrong point to use the scaler and it causes a bug in wrapper. 
The scaler must be fitted on the training data and then used during evaluation. Also the scaler must be embedded in the model and saved in the file (and obviously loaded in the wrapper). 
Please @apoman38 fix this bug.